### PR TITLE
Oliver test branch

### PR DESCRIPTION
**Pull request by JakeDaBoss**
Major fixes across the board.

More in the commit history.

### CODE REVIEW

**Comment by JakeDaBoss** [comment]
@BlueNexus

File: baystation12.int [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 AUTO_FILE_DIR: OFF
 MAP_ICON_TYPE: 0
 LAST_COMPILE_VERSION: 510.1347
-WINDOW: code\game\objects\structures\crates_lockers\closets\secure\security.dm;code\game\objects\items\devices\radio\headset.dm
+WINDOW: icons\obj\decals.dmi;code\game\objects\structures\signs.dm;maps\exodus\exodus-0.dmm;maps\exodus\exodus-1.dmm;maps\exodus\exodus-2.dmm;maps\exodus\exodus-workbook.dmm;maps\torch\torch_elevator.dm;maps\exodus\exodus_elevator.dm;maps\exodus\exodus_areas.dm;maps\exodus\exodus_define.dm;code\modules\mob\mob.dm;code\modules\multiz\turf.dm;code\modules\turbolift\turbolift_map.dm;code\modules\turbolift\turbolift_areas.dm;code\modules\modular_computers\file_system\programs\file_browser.dm
+FILE: maps\exodus\exodus-0.dmm
 */
 // END_INTERNALS
```
Comment (BlueNexus) [inline baystation12.int:7]: Remove lines 3, 4, 8 and 9. This file should never be edited.

**Review by BlueNexus — CHANGES_REQUESTED** [review]

**Comment by BlueNexus** [comment]
You should also properly port changes, rather than committing them as your own.

**Comment by JakeDaBoss** [comment]
Not exactly sure how to pull singular commits into our repo.

**Comment by BlueNexus** [comment]
You would cherry-pick them, but I doubt you know how to do that. This will do for now, once the requested change is done.

**Comment by JakeDaBoss** [comment]
Done @BlueNexus

**Review by BlueNexus — APPROVED** [review]

**Pull request merged**